app/domain/regulations.py

```diff
@@ -82,7 +82,8 @@ def compute_regulations(
             work_days_over_current_past_and_next_days,
             tz=user_timezone,
         )
-        if activity_to_compute_in_day(
+        # Do not mark empty previous day as computed
+        if index != 0 or activity_to_compute_in_day(
```

Comment:
Add info in `app/README_init_regulation_alerts.md` to remove `index != 0` when using command to init